src/Filter/modules/DoubleMultiSelectFilter/DoubleMultiSelectFilter.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+    DoubleMultiSelectFilterProps,
+    DoubleMultiSelectValues,
+    DoubleMultiSelectFilterOptions,
+    FilterModule
+} from 'src/Filter/types';
+import DoubleMultiSelect from './DoubleMultiSelect';
+import { FilterSectionHeader } from 'src/Filter/components';
+import FilterValueChips from 'src/Filter/components/FilterSectionHeader/FilterValueChips';
+import { FilterBadge } from 'src/Filter/components/FilterSectionHeader/filterSectionHeaderStyles';
+import styled from '@emotion/styled';
+
+const FilterCount = styled.div(({ theme }) => ({
```

Comment:
Can you add these styles to it's own style utils to be inline to other filters (thank @zach-jones-slalom for this one) 😂

Reply:
Sure, I don't usually for only a few lines of styles but might as well be consistent and keep the file focused.

Reply:
Done.